app/models/review.rb

```diff
@@ -1,11 +1,33 @@
 class Review < ActiveRecord::Base
   RATING = [1, 2, 3, 4, 5]
+  @score = 0
 
   belongs_to :neighborhood
   belongs_to :user
+  has_many :votes
 
   validates :rating, presence: true, inclusion: { in: RATING }
   validates :description, presence: true, length: { maximum: 20 }
   validates :user, presence: true
   validates :neighborhood, presence: true
+
+  def find_vote(user)
+    votes.find_by(user: user)
+  end
+
+  def voted_on_by?(user)
+    votes.find_by(user_id: user)
+  end
+
+  def upvoted_by?(user)
+    votes.find_by(user_id: user, value: 1)
+  end
+
+  def downvoted_by?(user)
+    votes.find_by(user_id: user, value: -1)
+  end
+
+  def total_votes(score)
```

Comment:
Unused method argument - `score`. If it's necessary, use `_` or `_score` as an argument name to indicate that it won't be used. You can also write as `total_votes(*)` if you want the method to accept any arguments but don't care about them.